### PR TITLE
Docs: adding jsfiddle snippet to DLL guide.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: adding a live jsfiddle snippet to DLL guide.

---

### Additional information

_Closes cksource/ckeditor5-internal#921_

### CODE REVIEW

File: docs/builds/guides/development/dll-builds.md [hunk 1]
```diff
@@ -149,6 +149,12 @@ For example:
 		} );
 ```
 
+## Live implementation sample
+
+Presented below is a working sample editor using the DLL mechanism. Observe the source and then click **"Result"** to switch to the live view of the working CKEditor 5 instance.
+
+<iframe width="100%" height="600" src="//jsfiddle.net/ckeditor/mv05bweu/8/embedded/html,result/dark/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
```
Comment (pomek) [inline docs/builds/guides/development/dll-builds.md:156]: When we set the final content of the sample, we should set a current version as the base.

image

Then, the URL will not contain the `/8/` part.

**Review by pomek — COMMENTED** [review]
I like the idea, but I'm not sure about the sample's content.

Except for showing the editor's features, we could consider producing the content that suggests going back from the sample to the documentation.

**Comment by FilipTokarski** [comment]
My two cents here - when we're ready with the content of the demo, it would be a nice finishing touch to adjust the iframe's height, so that there's no vertical scroll with the default content.

File: docs/builds/guides/development/dll-builds.md (hunk 1 above)
Comment (godai78) [reply docs/builds/guides/development/dll-builds.md]: Done.

**Pull request merged**